### DISCUSSION TITLE
怎样完整迁移皮肤站到新系统？

### DISCUSSION THREAD

**Discussion by nhtpr** [bs-community/blessing-skin-server#442] [Q&A]
正在使用6.0-beta4版本，php环境7.4.20，想升级皮肤站到6.02发行版，已经搭建了全新的php8.0.2虚拟机，请问怎样将beta版的数据完整迁移到新系统内？

**Comment by Big-Cake-jpg** [comment]
直接打个压缩包下载下来，数据库导出 sql 文件下载丢到新机器上不就可以了吗，何必在这里开个 Discussion 呢
